src/game/main.h

```diff
@@ -9,24 +9,31 @@ enum VIModes {
     MODE_PAL,
 };
 
-#define THREAD1_STACK 0x100
-#define THREAD2_STACK 0x800
-#define THREAD3_STACK 0x200
+#define THREAD1_STACK 0x0100
+#define THREAD2_STACK 0x0000
+#define THREAD3_STACK 0x0200
 #define THREAD4_STACK 0x2000
 #define THREAD5_STACK 0x2000
-#define THREAD6_STACK 0x400
+#define THREAD6_STACK 0x0400
+
+#define THREAD1000_STACK 0x0400
 
 enum ThreadID {
     THREAD_0,
     THREAD_1_IDLE,
-    THREAD_2_CRASH_SCREEN,
+    THREAD_2,
     THREAD_3_MAIN,
     THREAD_4_SOUND,
     THREAD_5_GAME_LOOP,
     THREAD_6_RUMBLE,
     THREAD_7_HVQM,
     THREAD_8_TIMEKEEPER,
     THREAD_9_DA_COUNTER,
+    NUM_THREADS,
+
+    THREAD_1000_CRASH_SCREEN_0 = 1000,
+    THREAD_1001_CRASH_SCREEN_1,
+    THREAD_1002_CRASH_SCREEN_2,
```

Comment:
What is going on here?

Reply:
If crash screen crashes on thread 1000, then the crash screen operates on itself from thread 1001, etc.

Recursive crash screens :)

Reply:
What is thread2 then?

Reply:
Thread ID 2 is unused now.

Reply:
Probably wants to be labeled as such (with stack entries and what not updated)